### DISCUSSION TITLE
qutebrowser crashed in android termux debian

### DISCUSSION THREAD

**Discussion by Chuanhuan** [qutebrowser/qutebrowser#6510] [Q&A] [answered]
I use proot to run debian and my default browser, chromium  `--no-sandbox`, run pretty well. 
In qutebrowser, I can `export QTWEBENGINE_DISABLE_SANDBOX=1` to skip error message.
However, qutebrowser need to access `/proc` information which is not allowed in no-root android phone.
Is any way to disable qutebrowser  read `/proc/sys/fs/inotify/max_user_watches `
my error message as below

```
 ⚡ root@localhost  ~  qutebrowser -R 
14:59:52 WARNING: QStandardPaths: XDG_RUNTIME_DIR not set, defaulting to '/tmp/runtime-root'
14:59:53 WARNING: QStandardPaths: XDG_RUNTIME_DIR not set, defaulting to '/tmp/runtime-root'
14:59:54 INFO: Run :adblock-update to get adblock lists.
14:59:54 INFO: Sandboxing disabled by user.
[14071:14123:0604/145955.426366:ERROR:file_path_watcher_linux.cc(71)] Failed to read /proc/sys/fs/inotify/max_user_watches
[14071:14132:0604/145955.455341:ERROR:udev_watcher.cc(50)] Failed to initialize a udev monitor.
14:59:55 ERROR: Fatal exception:
```

**Comment by The-Compiler** [comment, marked as answer]
Moving this to a discussion since I doubt this is a qutebrowser issue.

First of all, let me note that running without a sandbox is a security issue, and probably not something you should do for day-to-day browsing.

The

```
[14071:14123:0604/145955.426366:ERROR:file_path_watcher_linux.cc(71)] Failed to read /proc/sys/fs/inotify/max_user_watches
[14071:14132:0604/145955.455341:ERROR:udev_watcher.cc(50)] Failed to initialize a udev monitor.
```

is coming from the underlying Chromium. There's nothing qutebrowser can do about it, but it's probably not the reason for the crash.

Below it, there's `14:59:55 ERROR: Fatal exception:` and then your log cuts off. What's there (and also displayed in a message box if I remember correctly) would be much more relevant.
